src/power_grid_model_io/filters/__init__.py

```diff
@@ -58,3 +67,53 @@ def complex_inverse_imaginary_part(real: float, imag: float) -> float:
     Return the imaginary part of the inverse of a complex number
     """
     return (1.0 / (real + 1j * imag)).imag
+
+
+def get_winding(winding: str, neutral_grounding: bool = True) -> WindingType:
+    """
+    Return the winding type as an enum value, based on the string representation
+    """
+    winding_type = WINDING_TYPES[winding.upper()]
+    if not neutral_grounding:
```

Comment:
If it possible to have a `winding_type` with neutral like `wye_n` with `neutral_grounding = False?`

Reply:
I don't think so. I guess it's rudimentary code, written when z-winding wasn't supported yet. @TonyXiang8787 Can we remove this if statement (and thereby the neutral_grounding parameter)?

Reply:
It is possible to have winding `wye_n` but neutral is not grounded.

So we need this logic.

Reply:
ok